src/main/java/com/exadel/discountwebapp/history/enums/HistoryEnum.java

```diff
@@ -0,0 +1,13 @@
+package com.exadel.discountwebapp.history.enums;
+
+public enum HistoryEnum {
+    CATEGORY("com.exadel.discountwebapp.category.entity.Category"),
+    DISCOUNT("com.exadel.discountwebapp.discount.entity.Discount"),
+    VENDOR("com.exadel.discountwebapp.vendor.entity.Vendor");
+
+    public final String getClazz;
```

Comment:
1. can we use Class object instead of string?
2. getClazz is a bad name for a field. It should be a noun, for e.g. clazz or entityClass

Reply:
Changed to Class.
Renamed.